src/Server.js

```diff
@@ -109,18 +110,21 @@ async function serve(bundler, port, useHTTPS = false) {
     });
 
     server.once('listening', () => {
-      let addon =
-        server.address().port !== port
-          ? `- ${logger.chalk.yellow(
-              `configured port ${port} could not be used.`
-            )}`
-          : '';
-
-      logger.persistent(
-        `Server running at ${logger.chalk.cyan(
-          `${useHTTPS ? 'https' : 'http'}://localhost:${server.address().port}`
-        )} ${addon}`
-      );
+      const _port = server.address().port;
+      const protocol = useHTTPS ? 'https' : 'http';
+      const localAddr = `${logger.chalk.cyan(
+        `${protocol}://localhost:${_port}`
+      )}`;
+      const localIpAddr = `${logger.chalk.cyan(
+        `${protocol}://${ip.address()}:${_port}`
+      )}`;
+
+      logger.persistent('Server running at');
+      logger.persistent(`Local:             ${localAddr}`);
+      logger.persistent(`On Your Network:   ${localIpAddr}`);
+      if (_port !== port) {
+        logger.persistent(`${logger.chalk.yellow(`configured port ${port} could not be used.`)}`);
```

Comment:
Moved this into an if statement, otherwise it calls logger.persistent even without anything to print